File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoCase/src/pato/Voar_Foguete.java
package pato;

public class Voar_Foguete implements Padrao_Voaveis{
	
	private double velocidade;
	
	public Voar_Foguete()
	{
			
	}
	
	public String voar() {
		return "Voando como um foguete. Velocidade: " + getVelocidade();		
	}

	public double getVelocidade() {
		return velocidade = 1000;
	}	
}

File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoC#/Pato/VoarFoguete.cs
using Pato;
using System;
using System.Collections.Generic;
using System.Text;

namespace JogoPato
{
    public class VoarFoguete : IPadraoVoaveis
    {
		private double velocidade;

		public VoarFoguete()
		{
			velocidade = 1000;
		}

		public String voar()
		{
			return "Voando como um foguete. Velocidade: " + getVelocidade();
		}

		public double getVelocidade()
		{
			return velocidade;
		}
	}
}
